mmi/runner.py

```diff
@@ -316,7 +314,8 @@ def runner(arguments, wrapper_kwargs={}, extra_metadata={}):
             engine=arguments['<engine>'],
             configfile=arguments['<configfile>'],
             **wrapper_kwargs)
-    
+    # don't forget to initalize
+    model.initialize()
     # for replying to grid requests
```

Comment:
With the current implementation it will start gathering variables (with -g) before it will start processing incoming requests, so it needs to be initialised.